feature-libs/customer-ticketing/core/index.ts

```diff
@@ -1,3 +1,5 @@
+export * from '../root/config/customer-ticketing-config';
```

Comment:
Why is this being exported here in the core?

This is already being exported in `feature-libs/customer-ticketing/root/config/index.ts`

If ever it's needed to use something from that config, you would import the config as import { something } from '@customer-ticketing/root'